src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -147,6 +145,8 @@ private RowMapBuffer getTransactionRows(BinlogConnectorEvent beginEvent) throws
 		BinlogConnectorEvent event;
 		RowMapBuffer buffer = new RowMapBuffer(MAX_TX_ELEMENTS);
 
+		String temporaryQuery = "";
```

Comment:
nit: maybe rename this `currentQuery`, since it's holding the query for the current update?

Reply:
You have right, `currentQuery` is much better.